### PR TITLE
Integer BDD Scenarios

### PR DESCRIPTION
**Pull request by miguelcleverti**
Following up the integer BDD scenarios form dropped PR #211 

Todo:
- [x] Implementation

### CODE REVIEW

**Comment by pcardiga** [comment]
ping @bdunogier, @dpobel, @andrerom

File: Features/Stories/FieldTypes/IntegerFieldType.feature [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    Scenario: When editing a Content the label of an integer field must have the same name than field type from the respective Content Type
+        Given a Content Type with an "integer" with field definition name "Quantity" exists
+        When I create a content of this type
+        Then I should see a "Quantity" label related with the "integer" field
+
+    @javascript
+    Scenario: The label of an mandatory integer field of a Content must be marked as mandatory
+        Given a Content Type with a required "integer" with field definition name "Quantity" exists
+        When I create a content of this type
+        Then the "Quantity" field should be marked as mandatory
```
Comment (andrerom) [inline Features/Stories/FieldTypes/IntegerFieldType.feature]: do we call it mandatory, or do we call it required?

File: Features/Stories/FieldTypes/IntegerFieldType.feature [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+        ##
+        # Creating Content using a Content Type that has an Integer Field Type
+        ##
+    @javascript
+    Scenario: Publishing a valid integer Field works
```
Comment (andrerom) [inline Features/Stories/FieldTypes/IntegerFieldType.feature]: not needed I think as it is covered by the next one

File: Features/Stories/FieldTypes/IntegerFieldType.feature [hunk 3]
```diff
@@ -0,0 +1,202 @@
+Feature: Test the validations done on fields from PlatformUI - Integer fieldtype
+    In order to validate the integer fieldtype
+    As an Editor  user
+    I need to be able to create, update and delete content with integer fieldtypes
```
Comment (andrerom) [inline Features/Stories/FieldTypes/IntegerFieldType.feature]: deletion is not covered here, and neither should it as long as there is no logic around integer that needs to be verified on delete.

File: Features/Stories/FieldTypes/IntegerFieldType.feature (hunk 2 above)
Comment (andrerom) [inline Features/Stories/FieldTypes/IntegerFieldType.feature]: Publishing -> Creating, given you use Updating below that strictly also implies publishing

**Comment by pcardiga** [comment]
ping @bdunogier, @andrerom

**Comment by andrerom** [comment]
+1

But for other field types we don't need to test everything, it is cool that we do it on one to be able to test FieldType features in general as well, but as said in https://github.com/ezsystems/PlatformUIBundle/pull/243#issuecomment-119960457 we should right now focus on generic use cases to cover all features widely, as opposed to deeply and build up backlog of what to do next to have deeper coverage.

**Comment by bdunogier** [comment]
+1

**Comment by miguelcleverti** [comment]
@andrerom can this be merged?

**Comment by andrerom** [comment]
According to todo implement is not done, if you mean it is ready to be merged there should not be a open sub task on the PR ;)

**Comment by miguelcleverti** [comment]
@andrerom my mistake the implementation has been merged for some time now.

**Comment by andrerom** [comment]
+1 :)

**Pull request merged**